### ISSUE TITLE
Fix Windows build

### ISSUE THREAD

**Issue by aslakhellesoy** [cucumber/cucumber-ruby#1215]
It's currently failing: https://ci.appveyor.com/project/cucumberbdd/cucumber-ruby

**Comment by bv** [comment]
@aslakhellesoy, do you believe there is still anything to be fixed?
Currently

- the  #916 is closed
- am able to `bundle exec rake` on Windows with no issues
- No failures here https://ci.appveyor.com/project/cucumberbdd/cucumber-ruby/build/1.0.492

Thanks.

**Comment by brasmusson** [comment]
There are 97 scenarios in the feature files in the `features` directory with the tag `@todo-windows`, which I think means that the scenario is not executed on Windows.

**Comment by bv** [comment]
thank you Björn, now I got the idea, and can reproduce failing 77 scenarios out of 97.

**Comment by jaysonesmith** [comment]
@bv Taking a look through issues: Are you still working on this?

**Comment by bv** [comment]
@jaysonesmith trying to figure out why it is special about win platform

**Comment by jaysonesmith** [comment]
Pasting a bit of what I've said in the committer-ruby channel on Slack here:

There's what seems to be a pretty common issue finding expected files (maybe easy-ish to sort out depending on the cause) but there's another, more complicated issue I think.

It seems like scenarios, or the files they create, are bleeding into other scenarios and causing them to fail.

[9:32 PM]
I've spent a good amount of time going through scenarios marked `todo-windows` and running them to see what works/doesn't.

[9:33 PM]
Unfortunately, things that pass on their own may not when one or more other scenarios from the same file are run OR scenarios from other features are. On top of that, some situations can cause the number of scenarios that fail to differ.

[9:33 PM]
For instance: `bundle exec cucumber --profile windows_mri --order random features\docs\defining_steps\`

[9:34 PM]
On multiple, back to back runs, I could see 8 failures, then 7, then 9, then 8, and then 10.

[9:37 PM]
On the last run of 10 failures, here's what failed:

```skip_scenario.feature:3
nested_steps.feature:15
nested_steps_with_second_arg.feature:37
nested_steps_i18n.feature:17
nested_steps.feature:34
nested_steps.feature:80
nested_steps_with_second_arg.feature:9
ambiguous_steps.feature:55
nested_steps.feature:51
```

[9:37 PM]
In running each of those scenarios individually, every single one passed.

[9:38 PM]
If I do: `bundle exec cucumber -p windows_mri features/docs/defining_steps/skip_scenario.feature:3 features/docs/defining_steps/nested_steps.feature:15` the second scenario fails.

[9:38 PM]
If I switch their places, the latter scenario again fails

**Comment by xtrasimplicity** [comment]
Reopened, as this is still an issue.

**Comment by luke-hill** [comment]
Just as an update. I've spent some time today/yesterday digging through these. The Failures on the regular 2.3 build are now much fewer and far between. Only 3 regular failures and all for the same reason (Invalid line refs).

On 2.3-x64 about 40% fail. I suggest we should probably disable AppVeyor as we're ignoring it for the time being anyways. And only opt to reintroduce this once we've got it passing/stable?

**Comment by jaysonesmith** [comment]
I'd be fine with that for now. (Just need to make sure that we get it back going)

**Comment by luke-hill** [comment]
@bv are you still working on this?

My 0.02c is we keep this issue open, and remove the AppVeyor runs in a small breakout PR (Linking to this issue), and then when someone has time / makes progress they can come back to this issue ticket and try fix them? Updating / closing this ticket if they succeed in doing so.

**Comment by bv** [comment]
> @bv are you still working on this?
> 
I recall I could not progress much last time I made an attempt to approach the issue.
Will try again taking into account your input regarding instability in failures.
Thanks,

**Comment by vincent-psarga** [comment]
Fixed by #1370
